generators/service-hypersecure-dbaas-mongodb/index.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+const BaseGenerator = require('../lib/generatorbase');
+const SCAFFOLDER_PROJECT_PROPERTY_NAME = "hypersecuredb";
```

Comment:
scaffolder property = `hypersecuredb`

Reply:
@gconan  , it is already set to hypersecuredb 
SCAFFOLDER_PROJECT_PROPERTY_NAME = "hypersecuredb"; 

Is there another change that i am not seeing ?